charts/ratify/values.yaml

```diff
@@ -40,3 +40,7 @@ akvCertConfig:
   certName:
   tenantId:
   aadPodIdentity:
+oras:
+  authProviders:
+    azurewi: false
```

Comment:
suggestion to use full name  "azureWorkloadIdentity" as its easier to understand for customers not familiar with wi.

Reply:
on the other hand, azurewi is consistent with the name of of the provider, I think its ok to use the shorthand

Reply:
anyway, your call :)

Reply:
had a look at other configs, should we added "enabled" to field name?  azureWorkloadIdentityEnabled, enabled suggest a value of true or false.

Reply:
k8secretsEnabled

Reply:
sure just updated it